episodes/severity-static.Rmd

```diff
@@ -0,0 +1,640 @@
+
+```{r}
+# Load the Ebola 1976 data provided with the package
+data("ebola1976")
+
+# view top data
+ebola1976 %>% as_tibble()
+```
+
+When we use `cfr_static()` to `data` only, we calculate the naive CFR:
+
+```{r}
+# Calculate the static CFR without adjusting for delays
+cfr_static(data = ebola1976)
+```
+
+:::::::::::::::::::::::::::::::::::::::: challenge
+
+Load the `ebola_kikwit_1995` from the `{outbreaks}` package and estimate the naive CFR.
+
+:::::::::::::::::::: hint
+
+Inspect the format of the data input.
+
+- Does it contain daily data?
+- Does the column names are as required by `cfr_static()`?
+- How would you rename column names from a data frame?
+
+::::::::::::::::::::
+
+:::::::::::::::::::: solution
+
+We can use `dplyr::rename()` to adapt the external data to fit the data input for `cfr_static()`.
+
+```{r}
+library(outbreaks)
+
+outbreaks::ebola_kikwit_1995 %>%
+  glimpse()
+
+outbreaks::ebola_kikwit_1995 %>%
+  dplyr::rename(
+    cases = onset,
+    deaths = death
+  ) %>%
+  cfr::cfr_static()
+```
+
+::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::::::
+
+## Biases that affect CFR estimation
+
+[Lipsitch et al., 2015](https://journals.plos.org/plosntds/article?id=10.1371/journal.pntd.0003846) describe two potential biases that can affect the estimation of CFR (and its possible solutions):
+
+### Preferential ascertainment of severe cases
+
+For diseases with a _spectrum_ of clinical presentation, those cases that come to the attention of public health authorities and registered into surveillance databases will typically be people with the most severe symptoms who seek medical care, are admitted to a hospital, or die. 
+
+Therefore, the CFR will typically be higher among _detected cases_ than among the entire population of cases, given that the latter may include individuals with mild, subclinical, and (under some definitions of “case”) asymptomatic presentations.
+
+### Bias due to delayed reporting of death
+
+During an _ongoing_ epidemic, there is a delay between the time someone dies and the time their death is reported. Therefore, at any moment in time, the list of cases includes people who will die and whose death has not yet occurred or has occurred but not yet been reported. Thus, dividing the cumulative number of reported deaths by the cumulative number of reported cases at any moment will underestimate the true CFR.
+
+The key determinants of the magnitude of the bias are the epidemic _growth rate_ and the _distribution of delays_ from case-reporting to death-reporting; the longer the delays and the faster the growth rate, the greater the bias.
+
+:::::::::::::::::::: instructor
+
+We can showcase this last bias using the [concept described in this `{cfr}` vignette](https://epiverse-trace.github.io/cfr/articles/cfr.html#concept-how-reporting-delays-bias-cfr-estimates).
+
+<!-- create code and then a .gif? -->
+
+::::::::::::::::::::
+
+:::::::::::::::::::: testimonial
```

Comment:
Should this be renamed to "Case study"?

Reply:
added a title instead in https://github.com/epiverse-trace/tutorials-middle/pull/1/commits/bfd2ec0a208c8a30b472e7ab99a06ecd1a946722, to avoid conflicting terminology with how we define [case studies](https://github.com/orgs/epiverse-trace/discussions/81)